packages/python-google-compute-engine/google_compute_engine/distro_lib/helpers.py

```diff
@@ -98,3 +124,18 @@ def CallNtpdate(logger):
     logger.warning('Failed to sync system time with ntp server.')
   else:
     logger.info('Synced system time with ntp server.')
+
+def CallWriteViaSysCtl(logger, name, value):
```

Comment:
weird function name imo

Reply:
Agreed - how about `ConfigureSysctl` or `CallSysctl`?

Reply:
renamed to CallSysctl